src/System.Private.CoreLib/src/System/Runtime/CompilerServices/ConditionalWeakTable.cs

```diff
@@ -317,8 +317,8 @@ private class Container
         {
             internal Container()
             {
-                _buckets = new int[0];
-                _entries = new Entry[0];
+                _buckets = Array.Empty<int>();
+                _entries = Array.Empty<Entry>();
```

Comment:
This is always used as `new Container().Resize()` - that allocates a bunch of unnecessary garbage. It would be better to just allocate the arrays of the right size here to avoid all the garbage instead of patching over it with `Array.Empty`.